docs/libertyExtensions.md

```diff
@@ -40,7 +40,7 @@ The following properties are supported for server configuration.
 | configFile| File | 2.0 | Location of the `server.xml` file used by the server instance. The default value is `/src/main/liberty/config/server.xml`. After the | No|
 | jvmOptions| List | 2.0 | Inline `List` of jvm options that is written to the jvm.options file in the server directory. These properties take precedence over a specified jvm.options file.| No|
 | jvmOptionsFile| File | 2.0 | Location of the file containing JVM options to copy to the jvm.options file in the server instance. The default value is `/src/main/liberty/config/jvm.options`.| No|
-| looseApplication | boolean | 2.0 | Indicates to install application using loose application configuration so that war files do not need to be regenerated for every application update. The default is `true`. | No |
+| looseApplication | boolean | 2.0 | Indicates to install application using loose application configuration so that war or ear files do not need to be regenerated for every application update. Loose Application is only supported by Gradle 4.0 or higher when generating loose ear files.  The default is `true`. | No |
```

Comment:
Instead of just documenting this requirement, can we add a gradle version check as soon as we see the loose app ear support is needed, and issue an error message stating that version 4.0 or later is required?

Reply:
Updated to throw an exception.

Reply:
This wording is bothering me. How about "The loose application support for ear files requires Gradle 4.0 or higher."